lib/run-in-terminal.coffee

```diff
@@ -233,6 +235,19 @@ class ArgumentsRequester
                 @save_and_hide()
                 start_terminal(@path or "", @line_edit_model.getText())
 
+switch require('os').platform()
+
+    when 'darwin'
+        default_Launchfile = """osascript -e 'tell application \"Terminal\"' -e 'activate' -e 'tell application \"Terminal\" to do script \"cd \\\"{working_directory}\\\" && {launcher} \\\"{file_path}\\\" \"' -e 'end tell'"""
+        default_Launchdir = """osascript -e 'tell application \"Terminal\"' -e 'activate' -e 'tell application \"Terminal\" to do script \"cd \\\"{working_directory}\\\" \"' -e 'end tell'"""
+
+    when 'win32'
+        default_Launchfile = 'start /D {working_directory} C:\Windows\System32\cmd.exe /u /k cd "{working_directory}" {launcher} "{file_path}"'
+        default_Launchdir = 'start /D {working_directory} C:\Windows\System32\cmd.exe /u /k cd "{working_directory}"'
```

Comment:
Ok, I wasn't able to test on windows before. Now I have, and updated the lines. Apparently the backslashes needed escaping as well.